tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -924,7 +924,7 @@ Polymer({
       const valueList: number[] = [];
       const ints = new FloatList();
       ints.setValueList(valueList);
-      feat.setInt64List(ints);
+      feat.setInt64List(ints as any);  // TODO(#2266): Preserves behavior.
```

Comment:
Is this change related to this PR?

Reply:
Yes, it is: the upgrade to Clutz causes this to be an error without the
any-cast. PR #2266 actually fixes it, but I didn’t want to introduce a
(potentially) functional change here.

The upgrade to Clutz is required by the upgrade to the Closure compiler,
which is required by the upgrade to `rules_closure`, which is required
to fix the `--incompatible_depset_union`s, which are errors in the new
Bazel version. It’s a bundle of fun.